README.adoc

```diff
@@ -6,6 +6,37 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Operators
+
+In a new terminal, navigate to codeready-toolchain/host-operator directory and execute the following commands
+```
+$ minishift start --profile toolchain
```

Comment:
why do we need to specify a profile here?

Reply:
in addition, the script works (for now) only with the specific names of the profiles: https://github.com/codeready-toolchain/toolchain-common#scripts

Reply:
@MatousJobanek I was able to run them both under the same profile, for instance run both member and host under a member profile or vice versa, so i assumed the profile name didn't matter. I will verify this though!

Reply:
@xcoulon I'm not entirely sure, the readme on both the member and host repositories specifies you have to. Will test it out and let you know if it's needed though!

Reply:
@xcoulon @MatousJobanek so the profile name does need to be either host or member. I've updated the doc to reflect this!